package.json

```diff
@@ -61,10 +62,18 @@
     "eslint-config-prettier": "^9.0.0",
     "eslint-plugin-header": "^3.1.1",
     "eslint-plugin-react": "^7.33.2",
+    "husky": "^9.0.11",
     "jest": "^29.7.0",
+    "lint-staged": "^15.2.2",
     "prettier": "3.0.3",
     "ts-jest": "^29.1.1",
     "ts-node": "^10.9.2",
     "typescript": "^5.2.2"
+  },
+  "lint-staged": {
+    "{,src/**/}*.{ts,tsx}": [
+      "eslint --fix",
```

Comment:
Can we re-use the `npm run lint:fix` command here to keep everything consistent between hooks & package.json commands?

Reply:
yes.

Because I was thinking of doing a git commit to perform a lint on the modified files, if you call our lint:fix it actually performs a lint on all the files. it depends on what you're used to, but if you want it to be that way, I'll change it again.

Reply:
Let's just stick with this version so it's only running on staged files